src/components/Filters/index.js

```diff
@@ -304,7 +308,7 @@ class Filters extends Component {
                   >
                     {map(
                       inputModels[this.state.edit.name].availableOperators,
-                      name => <option>{name}</option>
+                      name => <option key={name}>{name}</option>
```

Comment:
I think that this should also be added to `PineTypes/Enum.js`.

Reply:
Well spotted! 🐄